### ISSUE TITLE
Temperature should be able to be adjusted via GUI

### ISSUE THREAD

**Issue by danielenriquez59** [Slamsneider/SimpleHtmlChatAgents#1]

**Comment by Slamsneider** [comment]
Yes! - And so should `max_tokens` so you can adjust them both before each inference.

Note: Using gpt-3.5-turbo-16k cost more to use than gpt-3.5-turbo. When I work (in my 'other' client) I start out in the 'small' model and when I see the tokencount gets high i just switch to the 16k version. (gonna make that switch automagical) hehe

**Comment by danielenriquez59** [comment]
for what its worth, you inspired me! I started a Vue project [here](https://github.com/danielenriquez59/primevue-chatbot) where the temperature is pulled out.
